hudi-common/src/test/java/org/apache/hudi/common/testutils/reader/HoodieFileGroupReaderTestUtils.java

```diff
@@ -25,7 +25,7 @@
 import org.apache.hudi.common.engine.HoodieReaderContext;
 import org.apache.hudi.common.model.FileSlice;
 import org.apache.hudi.common.table.HoodieTableMetaClient;
-import org.apache.hudi.common.table.read.HoodieFileGroupReader;
+import org.apache.hudi.common.table.read.FileSliceReader;
```

Comment:
We can keep `FileGroupReader` name in this PR.  Let @linliu-code decide how to restructure the classes.

Reply:
ok